service/host.go

```diff
@@ -323,29 +304,34 @@ func (s *host) Stop(reason string, exitCode int) error {
 	return err
 }
 
-func (s *host) startModules() map[Module]error {
-	results := map[Module]error{}
+func (s *host) startModules() []error {
```

Comment:
so this only happens once, right? how much time are we saving be starting these in parallel?